package.json

```diff
@@ -2,6 +2,7 @@
   "name": "svglint",
   "version": "1.0.5",
   "description": "Linter for SVGs",
+  "type": "module",
   "main": "src/svglint.js",
```

Comment:
```suggestion
  "exports": "./src/svglint.js"
```

Reply:
See: https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c#how-can-i-move-my-commonjs-project-to-esm